cmd/wardenkms/health_check_response.go

```diff
@@ -0,0 +1,23 @@
+package main
+
+type HealthCheckResponse struct {
+	// The number of nodes that are online
+	Online uint `json:"online"`
+
+	// The total number of nodes
+	Total uint `json:"total"`
+
+	// The consensus threshold
+	Threshold uint8 `json:"threshold"`
+
+	// Node statuses
+	Nodes []NodeStatus `json:"nodes"`
+}
+
+type NodeStatus struct {
+	// The address of the node
+	Address string `json:"address"`
+
+	// The status of the node
+	Status string `json:"status"`
+}
```

Comment:
Let's follow Go conventions when documenting types or fields: https://go.dev/wiki/CodeReviewComments#comment-sentences.

```suggestion
type HealthCheckResponse struct {
	// Online is...
	Online uint `json:"online"`

	// Total is...
	Total uint `json:"total"`

	// Threshold is...
	Threshold uint8 `json:"threshold"`

	// Nodes...
	Nodes []NodeStatus `json:"nodes"`
}

type NodeStatus struct {
	// Address...
	Address string `json:"address"`

	// Status...
	Status string `json:"status"`
}
```